assets/images/Svg.js

```diff
@@ -62,5 +62,5 @@ module.exports = function(name, props) {
     className = "";
   }
   props = Object.assign({}, props, { className, src: svg[name] });
-  return React.createElement(InlineSVG, props);
+  return <InlineSVG {...props} />;
```

Comment:
So this is an idea. But you'd need to touch more of the codebase. How about converting this component from:

```js
module.exports = function(name, props) {
  // eslint-disable-line
  if (!svg[name]) {
    throw new Error("Unknown SVG: " + name);
  }
  let className = name;
  if (props && props.className) {
    className = `${name} ${props.className}`;
  }
  if (name === "subSettings") {
    className = "";
  }
  props = Object.assign({}, props, { className, src: svg[name] });
  return <InlineSVG {...props} />;
};
```

to
```js
module.exports = function({ name, ...props }) {
  // eslint-disable-line
  if (!svg[name]) {
    throw new Error("Unknown SVG: " + name);
  }
  let className = name;
  if (props && props.className) {
    className = `${name} ${props.className}`;
  }
  if (name === "subSettings") {
    className = "";
  }
  props = { ...props, { className, src: svg[name] } };
  return <InlineSVG {...{ ...props, className, src: svg[name] }} />;
};
```

We would need to modify the usage of `Svg` but then we could use it like a component as well and convert the usages like so

```js
// from this
Svg("some-name", { foo: bar });

// to this
<Svg name="some-name", foo={bar} />
```